cli/src/test/java/com/box/l10n/mojito/cli/command/DropExportCommandTest.java

```diff
@@ -32,24 +32,43 @@ public void export() throws Exception {
 
         Repository repository = createTestRepoUsingRepoService();
 
+        getL10nJCommander().run("push", "-r", repository.getName(),
+                "-s", getInputResourcesTestDir("source").getAbsolutePath());
+
+        // wait for stats to be updated
+        Thread.sleep(1000);
+
+        Page<Drop> findAllBefore = dropClient.getDrops(repository.getId(), null, null, null);
+
+        getL10nJCommander().run("drop-export", "-r", repository.getName());
+
+        Page<Drop> findAllAfter = dropClient.getDrops(repository.getId(), null, null, null);
+
+        assertEquals("A Drop must have been added", findAllBefore.getTotalElements() + 1, findAllAfter.getTotalElements());
+    }
+
+    @Test
+    public void exportFullyTranslated() throws Exception {
+
+        Repository repository = createTestRepoUsingRepoService();
+
         getL10nJCommander().run("push", "-r", repository.getName(),
                 "-s", getInputResourcesTestDir("source").getAbsolutePath());
 
         Asset asset = assetClient.getAssetByPathAndRepositoryId("source-xliff.xliff", repository.getId());
         importTranslations(asset.getId(), "source-xliff_", "fr-FR");
         importTranslations(asset.getId(), "source-xliff_", "ja-JP");
 
-        Asset asset2 = assetClient.getAssetByPathAndRepositoryId("source2-xliff.xliff", repository.getId());
-        importTranslations(asset2.getId(), "source2-xliff_", "fr-FR");
-        importTranslations(asset2.getId(), "source2-xliff_", "ja-JP");
+        // wait for stats to be updated
+        Thread.sleep(1000);
```

Comment:
can you replace with `waitForCondition` please?

like in 

```
    waitForCondition("b4 should become untranslated when stats are computed", () -> {
            List<String> branches = repositoryClient.getBranches(repository.getId(), null, null, null,
                    false, false, null).stream().map(Branch::getName).sorted().collect(Collectors.toList());
            return branches.equals(asList("b4"));
        });
```

Reply:
See waitForRepositoryToHaveStringsForTranslations. Thank you!